document/CoreFormat.mdk

```diff
@@ -874,14 +878,14 @@ The value shall not be a JSON array or sub-object.
 
 ### General { #fix-General }
 
-A `fix` object represents a proposed fix for the issue represented by the `issue` object (&sect;[#issue]) in which it occurs.
+A `fix` object represents a proposed fix for the problem indicated by the `result` object (&sect;[#result]) in which it occurs.
```

Comment:
hm, I actually see occurs and occurred both in the doc. choose one and be consistent. :)